historical-contributors-agreement.txt

```diff
@@ -326,5 +326,6 @@ YYYY/MM/DD, github id, Full name, email
 2021/12/16, Ketler13, Oleksandr Martyshchenko, oleksandr.martyshchenko@gmail.com
 2021/12/25, Tinker1024, Tinker1024, tinker@huawei.com
 2021/12/31, Biswa96, Biswapriyo Nath, nathbappai@gmail.com
+2022/02/26, eirikur-grid, Eiríkur Fannar Torfason, eirikur [dot] torfason [at] gmail [dot] com
```

Comment:
this is no longer the correct model, please use GitHub DCO in your next commit

Reply:
Done